examples/typescript-angular/js/interfaces/ITodoScope.ts

```diff
@@ -10,7 +10,7 @@ module todos {
 		doneCount: number;
 		allChecked: boolean;
 		reverted: boolean;
-		statusFilter: { completed: boolean; };
+		statusFilter: any;
```

Comment:
Is there a way to specify this more narrowly? Not good with typescript, but could we mark `completed` as optional key?